.github/dependabot.yml

```diff
@@ -6,6 +6,8 @@ updates:
       interval: 'daily'
     ignore:
       - dependency-name: 'expo-*'
+      - dependency-name: 'react*'
```

Comment:
this will catch several packages that won't be updated by expo https://github.com/nearform/optic-expo/blob/master/package.json#L63-L71. it's not a big deal in itself, but nobody will ever update those packages then, which is potentially an issue. this is the compromise that's difficult to accept, and one reason why we haven't done this yet.

as an alternative, and smarter approach, if we could figure out (without hardcoding) what packages expo wants to manage itself, we could build an action which automatically closes PR opened by dependabot on those packages.

Reply:
Yeah, I added this general `react*` ignore inspired by the way it was done for the `expo-*` packages. But I understand and agree that it is not the best approach for the case. I'll close this PR and link your comment to the created issue, in order to give more context to it.